samples/interactive-tutorials/product/add_fulfillment_places.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START retail_add_fulfillment_places]
+import datetime
+import os
+import time
+
+from google.api_core.client_options import ClientOptions
+from google.cloud.retail import AddFulfillmentPlacesRequest, \
+    ProductServiceClient
+
+from setup.setup_cleanup import create_product, get_product
+
+project_number = os.getenv('GOOGLE_CLOUD_PROJECT_NUMBER')
+endpoint = "retail.googleapis.com"
+product_id = "add_fulfillment_test_product_id"
```

Comment:
Let's use a random id, which will make the test more stable.

Reply:
please take a look on my comment to samples/interactive-tutorials/product/remove_fulfillment_places.py